addons/account/views/account_portal_templates.xml

```diff
@@ -96,6 +96,7 @@
                     <div class="panel-body">
                         <t t-if="error or warning" t-call="account.portal_invoice_error"/>
                         <t t-if="success and (not error and not warning)" t-call="account.portal_invoice_success"/>
+                        <div id="payment_status"></div>
```

Comment:
I don't like having void div manipulated in inheriting modules. Couldn't you find a way to find this place without a void div ? Many bisous.